src/CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+macro(download_file_if_it_doesnt_exist filename url)
+	if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${filename})
+		message(STATUS "Downloading ${filename}")
+		file(DOWNLOAD ${url} ${CMAKE_CURRENT_SOURCE_DIR}/${filename} STATUS stat)
+		list(GET stat 0 success)
+		list(GET stat 1 error)
+		if (NOT ${success})
+			message(STATUS "Downloaded ${filename} succesfully")
+		else()
+			file(REMOVE ${CMAKE_CURRENT_SOURCE_DIR}/${filename})
+			message(FATAL_ERROR "Failed to download ${filename} (${error})")
+		endif()
+	endif()
+endmacro(download_file_if_it_doesnt_exist)
+
+download_file_if_it_doesnt_exist(ini.c https://raw.githubusercontent.com/benhoyt/inih/master/ini.c)
+download_file_if_it_doesnt_exist(ini.h https://raw.githubusercontent.com/benhoyt/inih/master/ini.h)
+download_file_if_it_doesnt_exist(lodepng/lodepng.c https://raw.githubusercontent.com/lvandeve/lodepng/master/lodepng.cpp)
+download_file_if_it_doesnt_exist(lodepng/lodepng.h https://raw.githubusercontent.com/lvandeve/lodepng/master/lodepng.h)
+download_file_if_it_doesnt_exist(dr_wav.c https://raw.githubusercontent.com/mackron/dr_libs/master/dr_wav.h)
```

Comment:
Probably it's better to specify a specific hash to prevent a breakage like [this](https://github.com/xtreme8000/BetterSpades/commit/85884b5fe7c970c1a0d2c547bcb9bdaaa221b9e3) (caused by [this](https://github.com/mackron/dr_libs/commit/4df41d95cf3bfa0d9a5a28faf6cf0d1be9d14b2d#diff-555c0d31bafccfc477ce783531da4f78R773)) in the future.